packages/firebase_ml_vision/lib/src/firebase_vision.dart

```diff
@@ -218,21 +207,17 @@ class FirebaseVisionImagePlaneMetadata {
 /// [rotation] defaults to [ImageRotation.rotation0]. Currently only rotates on
 /// Android.
 ///
-/// When using iOS, [rawFormat] and [planeData] throw [AssertionError] if
-/// `null`.
+/// When using iOS, [rawFormat] throws [AssertionError] if `null`,
+/// and [planeData] throws [AssertionError] if not empty.
 class FirebaseVisionImageMetadata {
   FirebaseVisionImageMetadata({
-    @required this.size,
-    @required this.rawFormat,
-    @required this.planeData,
+    required this.size,
+    required this.rawFormat,
+    required this.planeData,
```

Comment:
Should these be nullable? Since there's an assert for iOS

Reply:
Good catch, done